content-services/latest/support/index.md

```diff
@@ -22,7 +22,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 |  |  |
 | **Databases** | |
 | MySQL 8 | `mysql-connector-java-8.0.27.jar` |
-| MySQL 5.7.23 | `mysql-connector-java-5.1.40-bin.jar` |
+| MySQL 5.7.28 | `mysql-connector-java-5.1.49-bin.jar` |
```

Comment:
Good spot! I've only updated the content for ACS 7.2.1, and not back-ported to ACS 7.2.0 - waiting on a review from @damianujma to confirm this is the right thing to do.

Reply:
LGTM. I did the change only for ACS **7.2.1.** It will be better to ask @tiagosalvado10 because he worked on this issue [ACS-3271](https://alfresco.atlassian.net/browse/ACS-3271) for other ACS versions.

Reply:
I spoke to Tiago - no further changes needed in this PR so I'll resolve this conversation & merge the update.

Reply:
Thanks for the reviews @Amy-Hyland & @damianujma